sway-core/src/ir_generation/const_eval.rs

```diff
@@ -47,15 +46,8 @@ pub fn compile_const_decl(
                 TypedDeclaration::ConstantDeclaration(TypedConstantDeclaration {
                     name,
                     value,
-                    visibility,
-                }) => {
-                    // XXX Do we really only add public constants?
```

Comment:
This was a comment from when I copied the older AST->ASM code which only compiled pubs, and it has languished here for 6 months!  Thanks. 🙂